### DISCUSSION TITLE
12V AC measurement

### DISCUSSION THREAD

**Discussion by petermisovic** [OpenHantek/OpenHantek6022#183] [Q&A] [answered]
Hello guys, 
I'm trying to measure 12V AC with my new Hantek B6022BE using PP80B probe in 1X mode. I'm getting wave cut off on 5V (picture hantek-B6022BE-1x.png). So I changed my probe to 10X getting hantek-B6022BE-10X.png. How can I get measured AC voltage please? How can I measure it?

thanks,
Peter
![hantek-B6022BE-10X](https://user-images.githubusercontent.com/11147552/115215162-88618b80-a103-11eb-93dc-aad6b7e73089.png)
![hantek-B6022BE-1x](https://user-images.githubusercontent.com/11147552/115215165-88fa2200-a103-11eb-9ced-886bcb8a48e6.png)

**Comment by Vascom** [comment, marked as answer]
You need to set probe multiplier for CH110x at right up corner of the OpenHantek6022 window.

**Reply by petermisovic** [reply]
Hello Vascom, 
do you mean this setting please? 
![hantek-B6022BE-set10](https://user-images.githubusercontent.com/11147552/115218393-ce6c1e80-a106-11eb-955a-7a03bf8e5505.png)

**Reply by Vascom** [reply]
Yes.
Also you can try increase sample rate.

**Reply by petermisovic** [reply]
Hello Vascom,
I did it, thank you. 

Maybe I just do not understand the concept in general (from manual, or because of my knowledge). 

I have connected oscilloscope to 12V **AC**, probe 10X, probe multiplier set to x10. If I measure the voltage, I see 22V on CH1 
![20210419_131017](https://user-images.githubusercontent.com/11147552/115227304-9964c980-a110-11eb-8d59-599b45bc452a.png)

**Reply by Ho-Ro** [reply]
You measure the peak value Vpk that is not the RMS value VRMS that your multimeter would show. 
![image](https://user-images.githubusercontent.com/12542359/115229224-0ed19980-a113-11eb-83eb-7accbbd0d675.png)
Also your transformer delivers a higher voltage (e.g. 15V) if you do not have any load - the values on the label (e.g. 12V 1A) mean if you draw 1A then the output value is 12V.
Why don't you use the measured result in the bottom line. it calculates exactly the RMS value. 
![image](https://user-images.githubusercontent.com/12542359/115229759-b8188f80-a113-11eb-8f91-5ded70690c5e.png)

Please RTFM, it's only one mouse click away in the Help menu.

**Reply by petermisovic** [reply]
Hi Ho-Ro, I have read that (f) manual, but haven't found that info in Cursor Measurements. Need to go through again. Anyhow thank you for information, you helped me.

**Reply by Ho-Ro** [reply]
More info about voltage measurement can be found on page 6 (named Voltage). And please do yourself a favour and read about electrical engineering and metrology if you want to use a scope. Also knowledge about relationship between time domain (voltage) and frequency domain (spectrum) is important.

**Reply by petermisovic** [reply]
OK, of course.

**Comment by vizkoze** [comment]
Also try to adjust volts per division. The scope clips at + and - 5v . There is a voltage divider network at the input which is selected by the software. Set volts per division at 10V and see what happens.


________________________________

**Reply by Ho-Ro** [reply]
This doesn't change anything. The scope cannot measure more than +- 5 Volt at the input (without external divider).
The [manual](https://github.com/OpenHantek/OpenHantek6022/blob/main/docs/OpenHantek6022_User_Manual.pdf) tells this in the 1st page:
_The measurable input signal range is ±5 V._
It also shows and explains the frontend schematics on page 16.

**Reply by petermisovic** [reply]
"The measurable input signal range is ±5 V." yep, I read that and I'm little bit confused, as next stage tells about 35V as max. But yes, it requires knowledge about oscilloscope in general.

**Reply by Ho-Ro** [reply]
Value over +- 5V will be clipped, values over +-35V can damage your scope.

**Reply by petermisovic** [reply]
OK got it. So to see not clipped values higher than 5 (but lower then 35) I need to multiply. Then if I multiply using the probe by 10, corresponding setting should be set by xN voltage section, right?
